src/fromager/overrides.py

```diff
@@ -83,6 +83,18 @@ def patches_for_source_dir(
     return sorted((patches_dir / source_dir_name).glob("*.patch"))
 
 
+def patches_for_req(
+    patches_dir: pathlib.Path, req: str
```

Comment:
```suggestion
    patches_dir: pathlib.Path, req_name: str
```
since you just pass only the req name so maybe this var name might be better

Reply:
Done